rr_common/src/follower/follower.cpp

```diff
@@ -92,6 +106,8 @@ int main(int argc, char** argv) {
     nhp.param("setpoint_topic", setpoint_topic, string("/setpoint"));
     nhp.param("topic_from_controller", topic_from_controller, string("/control_effort"));
 
+    visonBox_pub = nhp.advertise<geometry_msgs::PolygonStamped>("/follower/vision_box", 1);
```

Comment:
If you are using `nhp`, ros includes the node name (in this case /follower) in front of the topic name if you don't include a slash (/) in front of the name. So just do `nhp.advertise<geometry_msgs::PolygonStamped>("vision_box", 1);` so that the topic name changes if we change the node name.